db/migrations/202408061737_add_boostagrams_and_use_json.go

```diff
@@ -0,0 +1,32 @@
+package migrations
+
+import (
+	_ "embed"
+
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+// This migration adds boostagram column to transactions
+var _202408061737_add_boostagrams_and_use_json = &gormigrate.Migration{
+	ID: "202408061737_add_boostagrams_and_use_json",
+	Migrate: func(tx *gorm.DB) error {
+		if err := tx.Exec("ALTER TABLE transactions ADD COLUMN boostagram JSONB").Error; err != nil {
+			return err
+		}
+
+		if err := tx.Exec(`
```

Comment:
can this be wrapped in a transaction (note: `tx` is not actually a transaction by default in gormigrate - see one of the recent migrations of how I did it)

Reply:
Thanks for updating the metadata column!

Reply:
> can this whole migration be wrapped in a transaction 

Is this because we don't want only one to succeed in case of failures?

Done btw 🙌

Reply:
We don't want the migration to half-succeed - it's hard to recover from